ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -163,7 +163,7 @@ public void deleteFastSyncState() {
   protected FastSyncState updateMaxTrailingPeers(final FastSyncState state) {
     if (state.getPivotBlockNumber().isPresent()) {
       trailingPeerRequirements =
-          Optional.of(new TrailingPeerRequirements(state.getPivotBlockNumber().getAsLong(), 0));
+          Optional.of(new TrailingPeerRequirements(state.getPivotBlockNumber().getAsLong(), 5));
```

Comment:
My suggestion is, we should probably eliminate this TrailingPeer validation but for test purpose we can easily skip it by assigning  Long.MAX_VALUE to the second param of the TrailingPeerRequirements constructor